crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -164,6 +164,7 @@ impl CheckpointExecutor {
                 &mut next_to_schedule,
                 epoch_store.clone(),
             );
+            self.metrics.checkpoint_exec_pending_transactions.set(pending.len() as u64);
```

Comment:
This will give you the number of checkpoint tasks pending execution, not number of transactions total (each checkpoint task will of course schedule all transactions in that checkpoint). Did you intend for this? If so, the naming is deceiving. If not, consider getting # transactions from `CheckpointExecutor::execute_transactions`

Reply:
Good point. I think the metrics from transaction manager should be good enough for the number of executing transactions. And the logic here makes it unnecessary to monitor the number of executing checkpoints. So just raising the concurrent checkpoint execution limit for now.